### DISCUSSION TITLE
Notches

### DISCUSSION THREAD

**Discussion by Lavanya-Bala** [Eomys/pyleecan#347] [Q&A]
Using the class notch to add stator notches creates them on the inner circle of the stator between stator slots. Is there any way to add the notches to the outer circle instead?

**Comment by BonneelP** [comment]
Hello,

For now we can only add notches to the "bore radius" of a lamination. The code architecture should enable to add this new feature without too much complexity. Here is the procedure to add notches on the "yoke radius": A new property "yoke_notch_list" (type: [Notch]) can be added to Lamination and then in the build_geometry of the Lamination daughters, instead of generating an Arc or a circle, the same procedure as for the bore radius is used to get the lines of the slots. 
The only problem I see is regarding the "get_Rbo" and "is_outwards" methods of the slot. One would need to (temporally) change is_internal of Lamination before calling the slot build_geometry to enforce the use of the correct radius. Do you think you would have time to work on this feature ? I yes I can help further help you. 

Best regards,
Pierre
Ps: https://pyleecan.org/slot.convention.html

**Reply by Lavanya-Bala** [reply]
Thanks for responding, Pierre. I will try to look into this and see if it is something I could do

**Comment by BonneelP** [comment]
Great :) I have open an issue on this topic with more in depth explanations/resources #352.
Feel free to ask if you need any help.

Best regards,
Pierre
